builders/interoperability/xcm/xcm-sdk/v1/xcm-sdk.md

```diff
@@ -695,4 +804,6 @@ The `fee` property returns the amount of fees to be paid along with information
     ```
 
 !!! note
-    For more information on assets and asset amounts, including fees, please refer to the [XCM SDK Reference](/builders/interoperability/xcm/xcm-sdk/v1/reference#assets){target=_blank}.
\ No newline at end of file
+    For more information on assets and asset amounts, including fees, please refer to the [XCM SDK Reference](/builders/interoperability/xcm/xcm-sdk/v1/reference#assets){target=_blank}.
+  
+--8<-- 'text/disclaimers/third-party-content.md'
```

Comment:
The XCM SDK page has a disclaimer but the reference page does not. Not sure if this is intentional.

Reply:
Yes it is, only cause the XCM SDK page uses third party signers like Ethers